### ISSUE TITLE
🧠 UAT — 2026-04-05 (16 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#56]
### ⭐ Spotlight

**1. [Structured Sparse Transition Matrices to Enable State Tracking in State-Space Models](https://openreview.net/forum?id=RDbuSCWhad)**
  *Aleksandar Terzic, Nicolas Menet, Michael Hersche et al.*
  📍 NeurIPS 2025 spotlight
  > Modern state-space models (SSMs) often utilize structured transition matrices
which enable efficient computation but pose restrictions on the model’s expressivity,
as measured in terms of the abili...

**2. [Learning to (Learn at Test Time): RNNs with Expressive Hidden States](https://openreview.net/forum?id=wXfuOj9C7L)**
  *Yu Sun, Xinhao Li, Karan Dalal et al.*
  📍 ICML 2025
  > Self-attention performs well in long context but has quadratic complexity. Existing RNN layers have linear complexity, but their performance in long context is limited by the expressive power of th...

**3. [Everything Everywhere All at Once: LLMs can In-Context Learn Multiple Tasks in Superposition](https://openreview.net/forum?id=5hZCK4Wbex)**
  *Zheyang Xiong, Ziyang Cai, John Cooper et al.*
  📍 ICML 2025
  > Large Language Models (LLMs) have demonstrated remarkable in-context learning (ICL) capabilities. In this study, we explore a surprising phenomenon related to ICL: LLMs can perform multiple, comput...

**4. [Implicit Language Models are RNNs: Balancing Parallelization and Expressivity](https://openreview.net/forum?id=5EbiopWH6e)**
  *Mark Schöne, Babak Rahmani, Heiner Kremer et al.*
  📍 ICML 2025
  > State-space models (SSMs) and transformers dominate the language modeling landscape. However, they are constrained to a lower computational complexity than classical recurrent neural networks (RNNs...

**5. [Do We Really Need Message Passing in Brain Network Modeling?](https://openreview.net/forum?id=KRosBwvhDx)**
  *Liang Yang, Yuwei Liu, Jiaming Zhuo et al.*
  📍 ICML 2025
  > Brain network analysis plays a critical role in brain disease prediction and diagnosis. Graph mining tools have made remarkable progress. Graph neural networks (GNNs) and Transformers, which rely o...

**6. [Covered Forest: Fine-grained generalization analysis of graph neural networks](https://openreview.net/forum?id=xvLVYrYQ8a)**
  *Antonis Vasileiou, Ben Finkelshtein, Floris Geerts et al.*
  📍 ICML 2025
  > The expressive power of message-passing graph neural networks (MPNNs) is reasonably well understood, primarily through combinatorial techniques from graph isomorphism testing. However, MPNNs' gener...

### 📌 Poster

**7. [Towards Text-Mask Consistency in Medical Image Segmentation](https://openreview.net/forum?id=riOevy2RwZ)**
  *Jie Gui, HangTu, Wen Sha et al.*
  📍 ICLR 2026 Poster
  > Vision-language models for medical image segmentation often produce masks that conflict with the accompanying text, especially under multi-site/multi-lesion descriptions. We trace this failure to t...

**8. [Better NTK Conditioning: A Free Lunch from (ReLU) Nonlinear Activation in Wide Neural Networks](https://openreview.net/forum?id=8dTBydCMSr)**
  *Chaoyue Liu, Han Bi, Like Hui et al.*
  📍 NeurIPS 2025
  > Nonlinear activation functions are widely recognized for enhancing the expressivity of neural networks, which is the primary reason for their widespread implementation. In this work, we focus on Re...

**9. [IMPACT: Irregular Multi-Patch Adversarial Composition Based on Two‑Phase Optimization](https://openreview.net/forum?id=RsZv37DGka)**
  *Zenghui Yang, Xingquan Zuo, Hai Huang et al.*
  📍 NeurIPS 2025
  > Deep neural networks have become foundational in various applications but remain vulnerable to adversarial patch attacks. Crafting effective adversarial patches is inherently challenging due to the...

**10. [Reward-Aware Proto-Representations in Reinforcement Learning](https://openreview.net/forum?id=8WdkG7g6Az)**
  *Hon Tik Tse, Siddarth Chandrasekar, Marlos C. Machado*
  📍 NeurIPS 2025
  > In recent years, the successor representation (SR) has attracted increasing attention in reinforcement learning (RL), and it has been used to address some of its key challenges, such as exploration...

**11. [Capturing Individual Human Preferences with Reward Features](https://openreview.net/forum?id=TgCkj4uEPl)**
  *Andre Barreto, Vincent Dumoulin, Yiran Mao et al.*
  📍 NeurIPS 2025
  > Reinforcement learning from human feedback usually models preferences using a reward function that does not distinguish between people. We argue that this is unlikely to be a good design choice in ...

**12. [Mixture of Scope Experts at Test: Generalizing Deeper Graph Neural Networks with Shallow Variants](https://openreview.net/forum?id=swwAPbtU07)**
  *Gangda Deng, Hongkuan Zhou, Rajgopal Kannan et al.*
  📍 NeurIPS 2025
  > Heterophilous graphs, where dissimilar nodes tend to connect, pose a challenge for graph neural networks (GNNs). Increasing the GNN depth can expand the scope (i.e., receptive field), potentially f...

**13. [Learning normalized image densities via dual score matching](https://openreview.net/forum?id=wtYcS4kxpF)**
  *Florentin Guth, Zahra Kadkhodaie, Eero P Simoncelli*
  📍 NeurIPS 2025
  > Learning probability models from data is at the heart of many machine learning endeavors, but is notoriously difficult due to the curse of dimensionality. We introduce a new framework for learning ...

**14. [EditInfinity: Image Editing with Binary-Quantized Generative Models](https://openreview.net/forum?id=vhzqdxuSB3)**
  *Jiahuan Wang, Yuxin Chen, Jun Yu et al.*
  📍 NeurIPS 2025
  > Adapting pretrained diffusion-based generative models for text-driven image editing with negligible tuning overhead has demonstrated remarkable potential. A classical adaptation paradigm, as follow...

**15. [Towards Robust Zero-Shot Reinforcement Learning](https://openreview.net/forum?id=vbFZlM7nRK)**
  *Kexin ZHENG, Lauriane Teyssier, Yinan Zheng et al.*
  📍 NeurIPS 2025
  > The recent development of zero-shot reinforcement learning (RL) has opened a new avenue for learning pre-trained generalist policies that can adapt to arbitrary new tasks in a zero-shot manner. Whi...

**16. [Nonparametric Quantile Regression with ReLU-Activated Recurrent Neural Networks](https://openreview.net/forum?id=IGKluZvI6J)**
  *Hang Yu, Lyumin Wu, Wenxin Zhou et al.*
  📍 NeurIPS 2025
  > This paper investigates nonparametric quantile regression using recurrent neural networks (RNNs) and sparse recurrent neural networks (SRNNs) to approximate the conditional quantile function, which...
